Makefile

```diff
@@ -32,6 +32,30 @@ open-project: ## open project on github
 lint: ## check style using tox and flake8 for Python 2 and Python 3
 	$(IN_VENV) tox -e py27-lint && tox -e py34-lint
 
+uwsgi-rebuild-validation: ## rebuild uwsgi_config.yml kwalify schema against latest uwsgi master.
+	$(IN_VENV) python scripts/config_manage.py build_uwsgi_yaml
+
+tool-shed-config-validate: ## validate tool shed YAML configuration file
+	$(IN_VENV) python scripts/config_manage.py validate tool_shed
+
+tool-shed-config-convert: ## convert old style tool shed ini to yaml
+	$(IN_VENV) python scripts/config_manage.py convert tool_shed
+
+tool-shed-config-rebuild-sample: ## Rebuild sample tool shed yaml file from schema
+	$(IN_VENV) python scripts/config_manage.py build_sample_yaml tool_shed --add_comments
+
+reports-config-validate: ## validate reports YAML configuration file
+	$(IN_VENV) python scripts/config_manage.py validate reports
+
+reports-config-convert: ## convert old style reports ini to yaml
```

Comment:
This output above seems to be missing the `module: galaxy.webapps.reports.buildapp:uwsgi_app()` line which is in the yml.sample however even if I add it I am getting the following error when trying to use the yml config as reports.ini: 
```
ConfigParser.MissingSectionHeaderError: File contains no section headers.
file: /Users/marten/devel/git/galaxy/config/reports.ini, line: 1
'uwsgi:\n'
```

When I take the whole new config (in yml) and paste it to the reports.yml.sample (removing the reports.ini completely) it also does not work. (With vanilla sample config run the app runs ok.)